lib/examples.js

```diff
@@ -41,7 +41,7 @@ fs.readFile(file, function (err, contents) {
 })
 `}]
 
-module.exports = function (callback) {
+module.exports = (callback) => {
```

Comment:
Well, people still prefer using `() =>` syntax as it looks more like a function and less like PHP or Ruby array. If needed, I can change single-argument functions to not have parens.